src/EventListener/LogRevisionsListener.php

```diff
@@ -250,13 +265,13 @@ public function onFlush(OnFlushEventArgs $eventArgs): void
             //doctrine is fine deleting elements multiple times. We are not.
             $hash = $this->getHash($entity);
 
-            if (\in_array($hash, $processedEntities, true)) {
+            if (in_array($hash, $processedEntities)) {
```

Comment:
is there really need to remove third argument? as default value equals `false`

Reply:
Fixed